Driverless AI/get-started-with-open-source-custom-recipes-tutorial/get-started-with-open-source-custom-recipes-tutorial.md

```diff
@@ -29,7 +29,7 @@ You will need the following to be able to do this tutorial:
 
 - A Driverless AI environment that is version 1.7.0 or newer
 
-- Basic knowledge of Driverless AI or doing the [Automatic Machine Learning Introduction with Drivereless AI Test Drive](https://training.h2o.ai/products/tutorial-1a-automatic-machine-learning-introduction-with-driverless-ai) 
+- Basic knowledge of Driverless AI or doing the [Automatic Machine Learning Introduction with Driverless AI Test Drive](https://training.h2o.ai/products/tutorial-1a-automatic-machine-learning-introduction-with-driverless-ai) 
```

Comment:
Please update the tutorial title to: 

Automatic Machine Learning Introduction with Driverless AI

In other words, remove (Test Drive) from the title since that part was removed from the tutorial tittle the last update for Tutorial 1.